language/tools/move-unit-test/src/test_runner.rs

```diff
@@ -166,8 +169,8 @@ impl SharedTestingConfig {
             &[], // no ty args, at least for now
             &test_info.arguments,
             pipeline,
-            /* output_opt */ None,
             /* stepwise */ true,
+            self.verbose,
```

Comment:
Could you add a TODO in the `execute_via_move_vm` function to support collecting VM logs if the `verbose` flag is set as well?